signwriting_animation/data/data_loader.py

```diff
@@ -0,0 +1,123 @@
+import os
```

Comment:
you should run pylint and address issues.

Reply:
<img width="1196" alt="image" src="https://github.com/user-attachments/assets/66a20175-4484-4ef8-8575-0f8a1d3adf28" />